ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/common/validation/Validators.kt

```diff
@@ -165,3 +194,20 @@ fun <E> Validator<E>.Property<CiString?>.isEvseId() =
     this.validate(EvseIdConstraint()) {
         it == null || it.matches("(?i)[a-z]{2}\\*?[a-z\\d]{3}\\*?E[a-z\\d*]{1,31}".toRegex())
     }
+
+fun <E> Validator<E>.Property<BigDecimal?>.validate() = this.isGreaterThanOrEqualTo(BigDecimal.ZERO)
```

Comment:
```suggestion
fun <E> Validator<E>.Property<BigDecimal?>.isPositive() = this.isGreaterThanOrEqualTo(BigDecimal.ZERO)
```

Reply:
I cannot put the same name for `fun <E> Validator<E>.Property<BigDecimal?>.` and `fun <E> Validator<E>.Property<Int?>.` extension functions because they have the same signature, that is why I changed one of the names

Reply:
Maybe you can use `isPositiveBigDecimal` or `isPositiveBD` or something like that, because `validate` is too generic I think.

Reply:
Absolutly ! thanks mate